LBH.AdultSocialCare.Api/V1/UseCase/Payments/Concrete/CreateDraftPayRunUseCase.cs

```diff
@@ -27,79 +25,37 @@ public CreateDraftPayRunUseCase(IPayRunGateway payRunGateway, IQueueService payr
 
         public async Task CreateDraftPayRun(DraftPayRunCreationDomain draftPayRunCreationDomain)
         {
-            var recurringPayRunTypes = new[] { PayrunType.DirectPayments, PayrunType.ResidentialRecurring };
-            var unApprovedPayRunStatuses = new[] { PayrunStatus.Draft, PayrunStatus.InProgress, PayrunStatus.WaitingForReview, PayrunStatus.WaitingForApproval, PayrunStatus.Approved };
-            var releaseHoldPayRunTypes =
-                new[] { PayrunType.DirectPaymentsReleasedHolds, PayrunType.ResidentialReleasedHolds };
-            ValidateDraftPayRun(draftPayRunCreationDomain, releaseHoldPayRunTypes);
+            var allowedPayRunTypes = new[] { PayrunType.ResidentialRecurring, PayrunType.ResidentialReleasedHolds };
```

Comment:
Can you please remove other payrun types and sync with FE?

Reply:
Updated